django/db/backends/sqlite3/_functions.py

```diff
@@ -99,6 +81,28 @@ def register(connection):
     connection.create_aggregate("VAR_POP", 1, VarPop)
     connection.create_aggregate("VAR_SAMP", 1, VarSamp)
 
+    # Some math functions are enabled by default in SQLite 3.35+.
+    sql = "SELECT 1 FROM pragma_compile_options() WHERE compile_options = ? LIMIT 1"
+    if connection.execute(sql, ["ENABLE_MATH_FUNCTIONS"]).fetchone() is None:
```

Comment:
We don't really need param substitution here

```suggestion
    sql = "SELECT 1 FROM pragma_compile_options() WHERE compile_options = 'ENABLE_MATH_FUNCTIONS' LIMIT 1"
    if connection.execute(sql).fetchone() is None:
```

Reply:
We don't, true. IIRC I left it like this because it meant that I didn't have to wrap the SQL string over multiple lines to fit the 88 character width, making it more legible. (If you're adamant that this should be changed then we can do.)